googlemock/include/gmock/gmock-generated-matchers.h

```diff
@@ -994,6 +994,211 @@ UnorderedElementsAre(const T1& e1, const T2& e2, const T3& e3, const T4& e4,
+    typename internal::DecayArray<T8>::type> Args;
+  return internal::ContainsSequenceMatcher<Args>(
+      Args(e1, e2, e3, e4, e5, e6, e7, e8));
+}
+
+template <typename T1, typename T2, typename T3, typename T4, typename T5,
+          typename T6, typename T7, typename T8, typename T9>
+inline internal::ContainsSequenceMatcher<
+    ::testing::tuple<
+      typename internal::DecayArray<T1>::type,
+      typename internal::DecayArray<T2>::type,
+      typename internal::DecayArray<T3>::type,
+      typename internal::DecayArray<T4>::type,
+      typename internal::DecayArray<T5>::type,
+      typename internal::DecayArray<T6>::type,
+      typename internal::DecayArray<T7>::type,
+      typename internal::DecayArray<T8>::type,
+      typename internal::DecayArray<T9>::type> >
+ContainsSequence(const T1& e1, const T2& e2, const T3& e3, const T4& e4,
+                 const T5& e5, const T6& e6, const T7& e7, const T8& e8,
+                 const T9& e9) {
+  typedef ::testing::tuple<
+    typename internal::DecayArray<T1>::type,
+    typename internal::DecayArray<T2>::type,
+    typename internal::DecayArray<T3>::type,
+    typename internal::DecayArray<T4>::type,
+    typename internal::DecayArray<T5>::type,
+    typename internal::DecayArray<T6>::type,
+    typename internal::DecayArray<T7>::type,
+    typename internal::DecayArray<T8>::type,
+    typename internal::DecayArray<T9>::type> Args;
+  return internal::ContainsSequenceMatcher<Args>(
+      Args(e1, e2, e3, e4, e5, e6, e7, e8, e9));
+}
+
+template <typename T1, typename T2, typename T3, typename T4, typename T5,
+          typename T6, typename T7, typename T8, typename T9, typename T10>
+inline internal::ContainsSequenceMatcher<
+    ::testing::tuple<
+      typename internal::DecayArray<T1>::type,
+      typename internal::DecayArray<T2>::type,
+      typename internal::DecayArray<T3>::type,
+      typename internal::DecayArray<T4>::type,
+      typename internal::DecayArray<T5>::type,
+      typename internal::DecayArray<T6>::type,
+      typename internal::DecayArray<T7>::type,
+      typename internal::DecayArray<T8>::type,
+      typename internal::DecayArray<T9>::type,
+      typename internal::DecayArray<T10>::type> >
+ContainsSequence(const T1& e1, const T2& e2, const T3& e3, const T4& e4,
+                 const T5& e5, const T6& e6, const T7& e7, const T8& e8,
+                 const T9& e9, const T10& e10) {
+  typedef ::testing::tuple<
+    typename internal::DecayArray<T1>::type,
+    typename internal::DecayArray<T2>::type,
+    typename internal::DecayArray<T3>::type,
+    typename internal::DecayArray<T4>::type,
+    typename internal::DecayArray<T5>::type,
+    typename internal::DecayArray<T6>::type,
+    typename internal::DecayArray<T7>::type,
+    typename internal::DecayArray<T8>::type,
+    typename internal::DecayArray<T9>::type,
+    typename internal::DecayArray<T10>::type> Args;
+  return internal::ContainsSequenceMatcher<Args>(
```

Comment:
All of the variadic-simulation here comes from the autogeneration from a .pump script.
To do this properly, you'd need to change the .h.pump file, regenerate the .h from it, and then make a PR of that change.